launch/cli/endpoints.py

```diff
@@ -0,0 +1,50 @@
+import click
+from rich.console import Console
+from rich.table import Table
+
+from launch.cli.client import init_client
+from launch.model_endpoint import ModelEndpoint
+
+
+@click.group("endpoints")
+@click.pass_context
+def endpoints(ctx: click.Context):
+    """Endpoints is a wrapper around model endpoints in Scale Launch"""
+
+
+@click.pass_context
+@endpoints.command("list")
+@click.pass_context
+def list_endpoints(ctx: click.Context):
+    """List all of your Endpoints"""
+    client = init_client(ctx)
+
+    table = Table(
+        "Endpoint name",
+        "Metadata",
+        "Endpoint type",
+        title="Endpoints",
+        title_justify="left",
+    )
+
+    for servable_endpoint in client.list_model_endpoints():
+        table.add_row(
+            servable_endpoint.model_endpoint.name,
+            servable_endpoint.model_endpoint.metadata,
+            servable_endpoint.model_endpoint.endpoint_type,
+        )
+    console = Console()
+    console.print(table)
+
+
+@endpoints.command("delete")
+@click.argument("endpoint_name")
+@click.pass_context
+def delete_endpoint(ctx: click.Context, endpoint_name: str):
```

Comment:
Similarly, is it possible to create or edit endpoints through the CLI?

Reply:
I am actually thinking creating bundles and endpoints should be separate tickets. Because the scope will be bigger. 
We'd need to support `load_fn-s`, `classes` or `zip` files. As well as a lot of extra parameters.

For example what would an endpoint creation look like?
```
scale-launch endpoints create \
  --endpoint_name ...
  --model_bundle ...
  --cpus ...
  --memory ...
  --gpus ...
  --min_workers ...
  --max_workers ...
  --per_worker ...
  --gpu_type ...
  --endpoint_type ...
```

Options:
- using python client instead: the code will be more clean
- create a new `endpoin_conf.yaml`
- using default arguments

Reply:
FWIW the [ticket](https://app.shortcut.com/scaleai/story/406052) description did mention creation and editing. But if you think that if creation and editing via CLI are not really workflows that people will do, then we can defer this or create separate follow-up tickets.